src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -194,7 +190,13 @@ private function getDocBlockFromProperty(string $class, string $property): ?DocB
             return null;
         }
 
-        return $this->docBlockFactory->create($reflectionProperty, $this->contextFactory->createFromReflector($reflectionProperty->getDeclaringClass()));
+        try {
+            return $this->docBlockFactory->create($reflectionProperty, $this->contextFactory->createFromReflector($reflectionProperty->getDeclaringClass()));
+        } catch (\InvalidArgumentException $e) {
+            // No valid doc block
+        }
+
+        return null;
```

Comment:
You can return in the catch (same below)

Reply:
Was wondering ... I guess that a personal preference thing for me. I always feel it looks weird to not have a return at the end if the return type is not void.